website/content/glossary/_index.md

```diff
@@ -0,0 +1,115 @@
+
+## E
+
+#### **Eco-Efficiency**
+
+Goods and services that use fewer resources and create less waste and pollution.
+
+#### **Emissions**
+
+An umbrella term under which for example also greenhouse gasses are covered. But emissions include other factors like micro particles that cause pollution. Emissions are categorized into three scopes:
+Scope 1: direct emissions: the CO2 output of the vehicles owned by the company
+Scope 2: indirect emissions: the power consumption of a data center, that causes emissions by the production of electricity
+Scope 3: indirect emissions, that not directly can be changed: delivery chain caused CO2 pollution
+
+
+#### **Embodied (Carbon) Emissions**
+
+Embodied, sometimes referred to as Embedded, Carbon Emissions are all emissions caused during a good's production. 
+That can include anything from transportation, storage, heating, the production of other goods to finish this good and so on.
+
+#### **Environmental Sustainability**
+
+Sustainability is the concept of a social, economical and environmental activity that, in simple terms, leaves the ecosystem as it found it. 
+Environmental sustainability focuses on the ecological impact of our doings and how we can improve our actions to reduce or eliminate our negative impact on nature, earth, and finally humanity.
+
+## G
+
+#### **Green House Gas (GHG)**
+
+Any kind of gasses caused by nature or human activities that trap heat in the atmosphere. GHG includes carbon dioxide, methane, ozone and others. 
+These gasses are called greenhouse gas because they have the same effect as a greenhouse: the sunlight can enter the house and part is reflected back, but it can‘t leave the greenhouse fully because the roof prevents the heat from escaping. 
+This causes the inside of a greenhouse to get hotter and hotter, the same as what’s going on with the earth. See also: Emissions.
+
+## N
+
+#### **Net Zero**
+
+Stands for zero-carbon emission. This means that a company that causes carbon emissions needs to store or neutralize the same amount of carbon. 
+In the best case scenario, an organization doesn‘t cause carbon pollution at all.
+
+## O
+
+#### **Operational Carbon Emission**
+
+The amount of carbon emitted during the active in-use phase, typical examples are buildings like data centers, or (if measurable)these can be also smaller elements like servers. 
+Per definition, operational carbon emissions include the usage, management and maintenance.
+
+## P
+
+#### **Power Usage Effectiveness (PUE)**
+
+The PUE describes how efficiently a data center uses energy. It defines how much energy is used by the computing equipment vs the overall power consumption of the data center. 
+An ideal PUE is 1.0. It is also a global standard under the ISO/IEC 30134-2:2016.
+
+PUE is calculated either through:
+Total Facility Energy / IT Equipment Energy
+1 + Non-IT Facility Energy / IT Equipment Energy
+
```

Comment:
Do we want a link here to an example of somewhere that uses this?  Does GSF have something on this?